pkg/test/framework/components/istio/config.go

```diff
@@ -139,6 +139,10 @@ type Config struct {
 	CustomSidecarInjectorNamespace string
 }
 
+func (c *Config) IsIstiodEnabled() bool {
+	return c.Values["global.istiod.enabled"] == "true"
```

Comment:
```suggestion
	if c.Operator {
		return c.Values["global.istiod.enabled"] != "true"
	}
	return false
```

Values only contains overrides, so we have to rely on the fact that istiod is default on operator and is never set for helm.